enum.c

```diff
@@ -2824,30 +2840,22 @@ each_val_i(RB_BLOCK_CALL_FUNC_ARGLIST(i, p))
 
 /*
  *  call-seq:
- *     enum.each_entry { |obj| block }  -> enum
- *     enum.each_entry                  -> an_enumerator
+ *    each_entry {|element| ... } -> self
+ *    each_entry                  -> enumerator
  *
- *  Calls <i>block</i> once for each element in +self+, passing that
- *  element as a parameter, converting multiple values from yield to an
- *  array.
```

Comment:
This is important text that should be added back (the part about converting multiple values from yield to an array), as it is the main reason to use `each_entry` instead of `each`. An example of this usage is given in the `Usage` section near the bottom of this commit.

Reply:
Modified.  Please confirm.